CHANGELOG.md

```diff
@@ -42,6 +42,11 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 * (server) [#18162](https://github.com/cosmos/cosmos-sdk/pull/18162) Start gRPC & API server in standalone mode.
 * (baseapp) [#16581](https://github.com/cosmos/cosmos-sdk/pull/16581) Implement Optimistic Execution as an experimental feature (not enabled by default).
+* (client/keys) [#17639](https://github.com/cosmos/cosmos-sdk/pull/17639) Allows using and saving public keys encoded as base64
```

Comment:
What's all of this?

Reply:
@julienrbrt not sure tbh, they came in through mergify...

Reply:
@julienrbrt I fixed the changelog, let me know.